server/events/yaml/valid/repo_cfg.go

```diff
@@ -75,6 +75,7 @@ type Step struct {
 	StepName   string
 	ExtraArgs  []string
 	RunCommand string
+	Variable   string
```

Comment:
```suggestion
        // EnvVarName is the name of the
        // environment variable that should be set by this step.
	EnvVarName   string
```

Reply:
Also add
```
// EnvVarValue is the value to set EnvVarName to.
EnvVarValue string
```

Because I think user should be also be able to do
```
- env:
    name: MY_ENV_VAR
    value: hi
```
in addition to:
```
- env:
    name: MY_ENV_VAR
    command: echo hi
```

Reply:
Ok. Good idea.